File: Python/0005.py
nDays = int(input())
days = list(map(int, input().split()))

mark3 = 0
for day in days:
    if day % 2 == 1:
        print(day, end=" ")
        mark3 += 1
print()

mark4 = 0
for day in days:
    if day % 2 == 0:
        print(day, end=" ")
        mark4 += 1
print()

if mark4 >= mark3:
    print("YES")
else:
    print("NO")

File: C++/0005.cpp
#include <iostream>
using namespace std;


int main()
{
    int days[100], nDays, mark4 = 0, mark3 = 0;

    cin >> nDays;
    for (int i = 0; i < nDays; i++)
        cin >> days[i];

    for (int i = 0; i < nDays; i++)
        if (days[i] % 2 == 1)
        {
            cout << days[i] << " ";
            mark3++;
        }
    cout << endl;

    for (int i = 0; i < nDays; i++)
        if (days[i] % 2 == 0)
        {
            cout << days[i] << " ";
            mark4++;
        }
    cout << endl;

    cout << (mark4 >= mark3 ? "YES" : "NO") << endl;

    return 0;
}
